src/noggit/scripting/script_tex.cpp

```diff
@@ -39,9 +49,8 @@ namespace noggit {
           + std::string(" (in call to tex_set_alpha)")
           );
       }
-      auto& ts = _chunk->texture_set;
-      ts->create_temporary_alphamaps_if_needed();
-      ts->tmp_edit_values.get()->operator[](index)[_index] = value;
+        ts->create_temporary_alphamaps_if_needed();
+        ts->tmp_edit_values.get()->operator[](index)[_index] = value;
```

Comment:
```suggestion
      ts->create_temporary_alphamaps_if_needed();
      ts->tmp_edit_values.get()->operator[](index)[_index] = value;
```
or is indentation wonky in the diff preview?